tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -56,8 +57,13 @@
         HorizontalAlignment="Stretch"
         VerticalScrollBarVisibility="Auto"
         VerticalScrollMode="Enabled">
-        <Grid
-            MaxWidth="{ThemeResource MaxPageContentWidth}"
+        <!--  Setting min width here prevents text clipping with "Next Steps"  -->

```

Comment:
Not sure how useful this comment is (especially the min width part, since I don't think I see minWidth set?)

Reply:
I see, it's set in code behind. I think it's more useful to explain over there, instead of the comments here.

Reply:
This was a hold-back when I set the MinWidth in XAMl.  Moved the comment to the code-behind.